mscore/pm.cpp

```diff
@@ -18,6 +18,8 @@
 //  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 //=============================================================================
 
+#include <QtCore/qglobal.h>
```

Comment:
ok...i figured out that I just need to move include "musescore.h" from a few lines down below to up here.